unix/coverage.c

```diff
@@ -8,6 +14,130 @@
 #include "py/builtin.h"
 #include "py/emit.h"
 #include "py/formatfloat.h"
+#include "py/builtin.h"
+#include "py/stream.h"
+#include "py/mphal.h"
+#include "py/nlr.h"
+
+// stream testing object, used to return non-blocking errors
```

Comment:
It's not only for non-blocking errors, so best to remove that part of the comment.